bin/blessed-modifier-order.sh

```diff
@@ -26,12 +27,17 @@ usage() {
     echo "$0 DIR ..."
     echo "Modifies in place all the java source files found"
     echo "in the given directories so that all java language modifiers"
-    echo "are in the canonical order given by Modifier#toString()."
+    echo "are in the canonical order."
     echo "Tries to get it right even within javadoc comments,"
     echo "and even if the list of modifiers spans 2 lines."
     echo
     echo "See:"
-    echo "https://docs.oracle.com/javase/8/docs/api/java/lang/reflect/Modifier.html#toString-int-"
```

Comment:
Looking at the latest docs for Modifier#toString()
it also needs a corresponding update
https://download.java.net/java/early_access/jdk23/docs/api/java.base/java/lang/reflect/Modifier.html#toString(int)

and if that is done, then we can keep referring to its javadoc.

References to javadoc and jls both have problems of becoming stale over time. 
There is no way to reference the "latest" spec via a URL.

Reply:
> it also needs a corresponding update
> https://download.java.net/java/early_access/jdk23/docs/api/java.base/java/lang/reflect/Modifier.html#toString(int)

From the preceding discussion, it follows that it cannot be done: https://mail.openjdk.org/pipermail/core-libs-dev/2024-January/117381.html

Reply:
Thanks ... now I see ...

---

When I originally wrote this script I recall hitting some false positives - a price we pay for having this modify comments.  Don't blindly apply all suggestions!

---

I don't recall considering "default".

---

It seems we no longer have a single canonical reference for modifier order.
(or more generally, a well-maintained high quality Java program style guide ...)

Reply:
> Thanks ... now I see ...

FWIW, @jddarcy thinks we can still improve `Modifier.toString(int)`. See this comment and its parent thread: https://github.com/openjdk/jdk/pull/17239#discussion_r1441050656